eng/build.ps1

```diff
@@ -610,20 +610,36 @@ try {
     InstallDotNetSdk $global:_DotNetInstallDir "2.1.503"
   }
 
-  if ($bootstrap) {
-    $bootstrapDir = Make-BootstrapBuild -force32:$test32
+  try
+  {
+    if ($bootstrap) {
+      $bootstrapDir = Make-BootstrapBuild -force32:$test32
+    }
+  }
+  catch
+  {
+      echo "##vso[task.logissue type=error](NETCORE_ENGINEERING_TELEMETRY=Build) Build failed"
+      throw $_
   }
 
   if ($restore -or $build -or $rebuild -or $pack -or $sign -or $publish -or $testCoreClr) {
-    BuildSolution
+      BuildSolution
```

Comment:
📝 It looks like your editor is ignoring our .editorconfig from #34560.

https://github.com/dotnet/roslyn/blob/6a2605a2d054c5b3baff5e13d035db0386ae3eb7/.editorconfig#L29-L31